PukingPredator/Assets/Scripts/PlayerCamera.cs

```diff
@@ -4,16 +4,29 @@
 
 public class PlayerCamera : MonoBehaviour
 {
-    [SerializeField] private Transform player;
+    [SerializeField] private GameObject player;
+    [SerializeField] private Transform playerTransform;
+    [SerializeField] private float sensitivity;
+
+    private Vector3 finalOffset;
 
     void Start()
     {
-        if (player == null) player = GameObject.FindGameObjectsWithTag("Player")[0].GetComponent<Transform>();
+        Cursor.lockState = CursorLockMode.Locked;
+        if (player == null) player = GameObject.FindGameObjectsWithTag("Player")[0];
```

Comment:
swap to use null coalescing assignment, ie:
if (player == null) player = GameObject.FindGameObjectsWithTag("Player")[0];
...can be swapped to:
player ??= GameObject.FindGameObjectsWithTag("Player")[0];